lookout/style/common.py

```diff
@@ -98,3 +99,24 @@ def huge_progress_bar(sequence: Sequence, log: logging.Logger, get_iter_name: Ca
              now, " " * 11,
              "=" * 80,
              )
+
+
+def handle_input_arg(input_arg: Union[str, Iterable[str]],
```

Comment:
Ok, I see that cmd module was not added to sourced.ml.core. 
I think it is fine to copy-paste this util function here.